cobalt/media/player/web_media_player_impl.cc

```diff
@@ -270,6 +273,17 @@ void WebMediaPlayerImpl::LoadProgressive(
   DCHECK_EQ(task_runner_, base::SequencedTaskRunner::GetCurrentDefault());
 
   UMA_HISTOGRAM_ENUMERATION("Media.URLScheme", URLScheme(url), kMaxURLScheme);
+  base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
+  if (disable_progressive_playback_ ||
+      command_line->HasSwitch(browser::switches::kDisableProgressivePlayback)) {
```

Comment:
Is there a reason to check for both disable_progressive_playback_ and a command switch? Doesn't the command line switch set the boolean, or is there a case where only one is true?

Reply:
`disable_progressive_playback_` is set by H5vcc rather than the command line. They could both be true, but we only need one.